header.php

```diff
@@ -1,15 +1,8 @@
 <?php
+    require "php/auth.php";
```

Comment:
I think that auth.php should come after password.php, because here we also check like if the user wants to log out, etc. Not sure, though.